custom/ilsgateway/tasks.py

```diff
@@ -30,10 +31,10 @@
 @periodic_task(run_every=crontab(hour="4", minute="00", day_of_week="*"),
                queue='logistics_background_queue')
 def report_run_periodic_task():
-    report_run('ils-gateway')
+    report_run.delay('ils-gateway')
 
 
-@task(queue='logistics_background_queue', ignore_result=True)
+@serial_task('{domain}', queue='logistics_background_queue', max_retries=0, timeout=60 * 60 * 12)
```

Comment:
Usually, this task takes 1-2 hour but I made it much more to be sure that only one task is run at the time.